lighthouse-core/lib/page-functions.js

```diff
@@ -446,20 +446,55 @@ function wrapRequestIdleCallback(cpuSlowdownMultiplier) {
   };
 }
 
-const getNodeDetailsString = `function getNodeDetails(elem) {
+/**
+ * @param {HTMLElement} element
+ */
+function getNodeDetailsImpl(element) {
+  // This bookkeeping is for the FullPageScreenshot gatherer.
+  if (!window.__lighthouseNodesDontTouchOrAllVarianceGoesAway) {
```

Comment:
FWIW, I also don't understand what `__lighthouseNodesDontTouchOrAllVarianceGoesAway` is trying to communicate :) It might be worth switching to something more straightforward